src/Tests/dotnet-workload-install.Tests/GivenDotnetWorkloadInstall.cs

```diff
@@ -116,16 +116,63 @@ public void GivenWorkloadInstallItCanUpdateInstalledManifests()
         {
             var parseResult = Parser.GetWorkloadsInstance.Parse(new string[] { "dotnet", "workload", "install", "xamarin-android" });
             var manifestsToUpdate = new (ManifestId, ManifestVersion, ManifestVersion)[] { };
-            (_, var installManager, var installer, _, _) = GetTestInstallers(parseResult, manifestUpdates: manifestsToUpdate);
+            (_, var installManager, var installer, _, _, _) = GetTestInstallers(parseResult, manifestUpdates: manifestsToUpdate);
 
             installManager.InstallWorkloads(new List<WorkloadId>(), false); // Don't actually do any installs, just update manifests
 
             installer.InstalledManifests.Should().BeEquivalentTo(manifestsToUpdate);
         }
+		
+        [Fact]
+        public void GivenWorkloadInstallItCanDownloadToOfflineCache()
+        {
+            var mockWorkloadIds = new WorkloadId[] { new WorkloadId("xamarin-android") };
+            var cachePath = "mockCachePath";
+            var parseResult = Parser.GetWorkloadsInstance.Parse(new string[] { "dotnet", "workload", "install", "xamarin-android", "--download-to-cache", cachePath });
+            (_, var installManager, var installer, _, _, _) = GetTestInstallers(parseResult);
+
+            installManager.Execute();
+
+            // 8 android pack packages
+            installer.CachedPacks.Count.Should().Be(8);
+            installer.CachedPacks.Where(pack => pack.Id.Contains("Android")).Count().Should().Be(8);
```

Comment:
should also have a manifest here. The older manifest file thing is annoying but we do need that.